### ISSUE TITLE
Source code is missing final line feed without syntax highlighting

### ISSUE THREAD

**Issue by hwjsnc** [JuliaDebug/Cthulhu.jl#101]
In Cthulhu.jl version 1.3.0, when syntax highlighting is off, code is displayed without a trailing line feed when shown with `[S]`. An example session:

```
julia> using Cthulhu

julia> f(x) = 3
f (generic function with 1 method)

julia> @descend f(1)

│ ─ %-1  = invoke f(::Int64)::Core.Compiler.Const(3, false)
CodeInfo(
    @ REPL[2]:1 within `f'
1 ─     return 3
)
Select a call to descend into or ↩ to ascend. [q]uit. [b]ookmark.
Toggles: [o]ptimize, [w]arn, [d]ebuginfo, [s]yntax highlight for Source/LLVM/Native.
Show: [S]ource code, [A]ST, [L]LVM IR, [N]ative code
Actions: [E]dit source code, [R]evise and redisplay
Advanced: dump [P]arams cache.

  ↩
```

After pressing `s` to turn off syntax highlighting:

```
[ Info: Turned off syntax highlighter for LLVM and native code.
Select a call to descend into or ↩ to ascend. [q]uit. [b]ookmark.
Toggles: [o]ptimize, [w]arn, [d]ebuginfo, [s]yntax highlight for Source/LLVM/Native.
Show: [S]ource code, [A]ST, [L]LVM IR, [N]ative code
Actions: [E]dit source code, [R]evise and redisplay
Advanced: dump [P]arams cache.

  ↩
```

After pressing `S` to display the source code, note that the prompt starts on the same line as the code:

```
f(x) = 3Select a call to descend into or ↩ to ascend. [q]uit. [b]ookmark.
Toggles: [o]ptimize, [w]arn, [d]ebuginfo, [s]yntax highlight for Source/LLVM/Native.
Show: [S]ource code, [A]ST, [L]LVM IR, [N]ative code
Actions: [E]dit source code, [R]evise and redisplay
Advanced: dump [P]arams cache.

  ↩
```

The issue also occurs for function definitions spanning multiple lines.

The following patch fixes the issue:

```
diff --git a/src/codeview.jl b/src/codeview.jl
index 4f27046..6eced43 100644
--- a/src/codeview.jl
+++ b/src/codeview.jl
@@ -4,7 +4,7 @@ highlighter_exists(config::CthulhuConfig) =
 __init__() = CONFIG.enable_highlighter = highlighter_exists(CONFIG)
 
 function highlight(io, x, lexer, config::CthulhuConfig)
-    config.enable_highlighter || return print(io, x)
+    config.enable_highlighter || return println(io, x)
     if !highlighter_exists(config)
         @warn "Highlighter command $(config.highlighter.exec[1]) does not exist."
         return print(io, x)
```

If it is more convenient for the maintainers, I can submit a pull request as well.

**Comment by timholy** [comment]
A pull request would be great, thanks!

**Comment by hwjsnc** [comment]
Fixed with PR #103

Thanks!
